dogpile/cache/region.py

```diff
@@ -1053,7 +1064,12 @@ def async_creator(key, mutex):
                 # sort the keys, the idea is to prevent deadlocks.
                 # though haven't been able to simulate one anyway.
                 keys_to_get = sorted(mutexes)
+                start_time = time.time()
                 new_values = creator(*keys_to_get)
+                log.debug(
+                    'Generate cache in %.2f s. for key: %r',
+                    time.time() - start_time, orig_key
+                )
```

Comment:
**mike bayer** (zzzeek) wrote:

creator is against keys_to_get here.  I'll get this fixed up, thanks!